src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/task/v6/node/ProcessingNode.java

```diff
@@ -249,8 +253,8 @@ public void updateFinishedQuantity() {
     public void onCalculationFinished() {
         super.onCalculationFinished();
 
-        this.singleItemSetToRequire = requirements.getSingleItemRequirementSet(true);
-        this.singleFluidSetToRequire = requirements.getSingleFluidRequirementSet(true);
+        this.singleItemSetToRequire = new ItemStackList(requirements.getSingleItemRequirementSet(true));
```

Comment:
Was there a reason to not just also convert `singleItemSetToRequire` into an Arraylist?

Reply:
No idea, don't remember :)

My guess is that those are returned in getters and those getters are used elsewhere and I wanted to keep the change minimal to increase the chance of it happening